build.go

```diff
@@ -541,10 +541,20 @@ func buildSnap(target target) {
 	snaparch := goarch
 	if snaparch == "armhf" {
 		goarch = "arm"
-	}	
+	}
+	snapver := version
+	if strings.HasPrefix(snapver, "v") {
+		snapver = snapver[1:]
+	}
+	snapgrade := "devel"
+	if matched, _ := regexp.MatchString(`^\d+\.\d+\.\d+$`, snapver); matched {
+		snapgrade = "stable"
+	}
 	err = tmpl.Execute(f, map[string]string{
-		"Version": version,
-		"Architecture": snaparch})
+		"Version":      snapver,
+		"Architecture": snaparch,
+		"Grade":        snapgrade,
+	})
```

Comment:
I think something went wrong when you merged some remote branch in, as that was proposed in some PR but then got removed I think, so you should remove it.

The proto compiler might be going nuts as you're using the wrong version of it?